src/Button/index.tsx

```
@@ -109,215 +55,161 @@ const getButtonDimensions = (size: Size) => {
+  return (
+    <button
+      ref={ref}
+      type={type}
+      onClick={handleClick}
+      disabled={disabled}
+      style={buttonStyle}
+      data-testid={testID}
+      aria-label={ariaLabel || accessibilityLabel}
+      onMouseEnter={(e) => {
+        if (!disabled) {
+          e.currentTarget.style.backgroundColor = colors.backgroundColor + "E6"; // Add hover effect
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)